package.json

```diff
@@ -19,35 +31,25 @@
 		"moment": "^2.29.4",
 		"pullstate": "^1.24.0",
 		"react": "^18.2.0",
+		"react-csv": "^2.0.3",
+		"react-d3-cloud": "^1.0.1",
 		"react-dom": "^18.2.0",
+		"react-highlight-words": "^0.17.0",
+		"react-masonry-css": "^1.0.16",
+		"react-redux": "^7.2.6",
 		"react-router-dom": "5.3.4",
 		"react-scripts": "5.0.1",
 		"react-simple-maps": "^3.0.0",
 		"react-tooltip": "^4.4.3",
 		"react-transition-group": "^4.4.5",
+		"source-map-explorer": "^2.5.3",
 		"swiper": "^8.4.4",
-		"web-vitals": "^2.1.0",
-    "@dnd-kit/core": "^4.0.3",
-    "@dnd-kit/sortable": "^5.1.0",
-    "@emotion/react": "^11.5.0",
-    "@emotion/styled": "^11.3.0",
-    "@mui/icons-material": "^5.0.4",
-    "@mui/material": "^5.0.4",
-    "@mui/x-data-grid": "^5.1.0",
-    "@reduxjs/toolkit": "^1.6.2",
-    "chart.js": "^3.6.2",
-    "chartjs-plugin-datalabels": "^2.0.0",
-    "clsx": "^1.1.1",
-    "d3": "^7.0.0",
-    "react-csv": "^2.0.3",
-    "react-d3-cloud": "^1.0.1",
-    "react-highlight-words": "^0.17.0",
-    "react-masonry-css": "^1.0.16",
-    "react-redux": "^7.2.6"
+		"web-vitals": "^2.1.0"
 	},
 	"scripts": {
 		"start": "react-scripts start",
-		"build": "react-scripts build",
+		"build": "GENERATE_SOURCEMAP=true react-scripts build",
```

Comment:
@martinchristov generating a sourcemap will slow down the build in the CI and increase the size of the build (at the moment the sourcemap is 16MB!). I'm not sure when it's served, but if it's always served, that would probably slow down the page-loading time significantly.

I might be mistaken, but by default the sourcemap is [generated unless `GENERATE_SOURCEMAP=false`](https://create-react-app.dev/docs/advanced-configuration/).
The `pages.yml` workflow makes use of that to get rid of it for the production build